ios/Podfile

```diff
@@ -65,7 +65,8 @@ target 'Runner' do
   pod 'Flutter', :path => 'Flutter'
 
   # Plugin Pods
-
+  pod 'Firebase/DynamicLinks'
```

Comment:
I think that a dependency should be getting added automatically by the firebase_dynamic_links plugin, so probably you can remove this.

Reply:
This was from a Google video on adding firebase dynamic links to an app.  Maybe that info is obsolete. I'll try it without.

Reply:
At the one minute mark:  https://www.youtube.com/watch?v=iSC5ed6OowA&index=2&list=PLl-K7zZEsYLnJVX_0zbKytptZGugPIbJR&linkId=58112791

Reply:
maybe that video was for non-flutter apps and that's why they say to add the pod?  The flutter package does this automatically?

Reply:
I removed that line.